recipe/install-pkg.sh

```diff
@@ -1,21 +1,67 @@
 #!/bin/bash
-
 set -ex
 
+if [[ "$PKG_NAME" == "msvc-headers-libs" ]]; then
+    ################################################################
+    # create map of required components/urls for msvc-headers-libs #
+    ################################################################
+
+    # download utility to determine necessary download urls
+    curl -LO https://raw.githubusercontent.com/mstorsjo/msvc-wine/master/vsdownload.py
+    # patch it to display the urls in the log
+    patch -u < $RECIPE_DIR/patches/0001-show-payload-url.patch
+    # create temporary folder
```

Comment:
Add this as a source in `meta.yaml`?

Reply:
Fair point, though we'd have to download all of https://github.com/mstorsjo/msvc-wine; I was also aiming to have the script be more easily locally executable for debbuging purposes. In any case, I don't have a strong preference, so I'll give it a shot.

Reply:
>  though we'd have to download all of https://github.com/mstorsjo/msvc-wine;

Why? You only download vsdownload.py above

Reply:
Thanks. I see the linter is unhappy, I'll fix it in few minutes